File: RepoTransBench/source_projects/Python/nickstenning_honcho/public_tests/test_public_process.py
from honcho.process import Process

def test_public_process_init():
    p = Process("worker.3", "proc_cmd", env={"HELLO": "world"})
    assert p.name == "worker.3"
    assert p.cmd == "proc_cmd"
    assert p.env["HELLO"] == "world"

File: RepoTransBench/target_projects/Python/Rust/nickstenning_honcho/public_tests/test_public_process.rs
use crate::process::Process;
use std::collections::HashMap;

#[test]
fn test_public_process_init() {
    let mut env = HashMap::new();
    env.insert("HELLO".to_string(), "world".to_string());
    let p = Process::new("worker.3", "proc_cmd", Some(env.clone()));
    assert_eq!(p.name, "worker.3");
    assert_eq!(p.cmd, "proc_cmd");
    assert_eq!(p.env.get("HELLO").unwrap(), "world");
}